### ISSUE TITLE
Conflicting shortcut

### ISSUE THREAD

**Issue by CyrilleB79** [mesteranas/AITranslater-nvda-add-on#7]
The default gesture for Translates clipboard text is `NVDA+alt+c`. This conflicts with and overrides NVDA's native shortcut to read comments in Word or Excel.

Could you choose another gesture which does not conflict with NVDA's native ones? Thanks!

**Comment by mesteranas** [comment]
You can change the shortcut from nvda settings

**Comment by CyrilleB79** [comment]
Of course, gestures can be changed.

But using native NVDA gestures is usually considered bad practice because you force users to re-define a gesture, or worse, to investigate why NVDA's native feature X or Y does not work.

All the gestures that are defined in this add-on conflict with native NVDA gestures: `NVDA+alt+c`, `NVDA+alt+r` and `NVDA+alt+t`.

**Comment by mesteranas** [comment]
You can suggest me a shortcuts because I don't know
